sphinx_ncs_theme/sections/header.html

```diff
@@ -34,15 +34,9 @@
             aria-expanded="false"
           >
             <span class="d-none d-md-inline">nRF Connect SDK</span>
-            v1.8.0
+            versions
```

Comment:
note: the drop down should always show the version being browsed, is it the case now?

Reply:
Yes, it should be. This text is a placeholder that is displayed before the hosted versions file is loaded.